app/templates/index.hbs

```diff
@@ -33,6 +33,15 @@
           English Service
         </div>
       </div>
+      <div class="col-sm-4 group" {{action "viewEnglishService"}}>
```

Comment:
Clicking on this `div` would fire the `viewEnglishService` action, which we do not want. Since this is for "Coming soon," we probably don't want any actions at all.